ffmpeg_tools/validation.py

```diff
@@ -402,20 +431,30 @@ def validate_audio_sample_rates(
 
 def validate_video_codec_conversion(src_codec, dst_codec):
     codec = codecs.VideoCodec(src_codec)
+
     if dst_codec not in codec.get_supported_conversions():
         raise UnsupportedVideoCodecConversion(src_codec, dst_codec)
+
+    if codecs.VideoCodec(dst_codec).get_encoder() is None:
+        raise MissingVideoEncoder(video_codec=dst_codec)
```

Comment:
Validation shouldn't fail in case of target codec. It should assume, that parameter doesn't change, if not specified. Related to review of https://github.com/golemfactory/golem/pull/4954#pullrequestreview-326881496

Reply:
Actually the assumption after #4954 is still that both video codec and resolution are mandatory. See my answer to a similar comment above: https://github.com/golemfactory/ffmpeg-tools/pull/29#discussion_r354965334.